webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js

```diff
@@ -40,18 +46,35 @@ const TasksTablePage = ({ getBreadcrumbs, history, ...props }) => {
     }));
   };
 
-  const TaskSelectedAction = (id, name) => {
-    props.actionSelected(id, name, url);
+  const modalProps = {
+    CANCEL_SELECTED: useForemanModal({ id: CANCEL_SELECTED_CONFIRM_MODAL_ID }),
```

Comment:
I think it might be more readable like that but this was made this way so it will be consistent with the tasksActions
``` JS
 closeModal={modalProps.CANCEL.setModalClosed}
 action={tasksActions[CANCEL]}
```
Do you think we should make the `taskActions` different too? or maybe change just the `modalProps`?

Reply:
@MariaAga It makes sense to me to change both of them